manual/encryption.rst

```diff
@@ -47,24 +47,24 @@ Password Protection
   Password protection is distinct from encryption. This point is often
   misunderstood. A PDF file can be encrypted without being
   password-protected. The intent of PDF encryption was that there
-  would be two passwords: a *user password* and an *owner password*.
+  would be two passwords: an *user password* and an *owner password*.
```

Comment:
`a` is correct here, at least in American usage. It's phonetic, not lexical, and we use `a` before the `y` sound in `yoozer`.

Reply:
Same in British English. It's just that some people can't tell their consonants from their elbows.